src/createAdminAccountScript.ts

```diff
@@ -1,88 +1,114 @@
-// Step by step
-// 1. Connect DB
-console.log('Admin entered: ' + process.argv[2]);
-
 // Import functions
 
 import generator from 'generate-password';
-
-
 import nodemailer from 'nodemailer';
+
 import User from './models/User';
 import connectDatabase from './database';
 import { hashPassword } from './helpers/passwordHash';
+import { createJWT } from './helpers/jwtHelper';
+import Token from './models/Token';
+import { exit } from 'node:process';
+
 require('dotenv').config();
```

Comment:
Instead of doing this, we can require it in the CLI. `ts-node -r dotenv/config src/createAdminAccountScript.ts` would work